pkg/openshift/rosa/accountroles.go

```diff
@@ -9,10 +9,15 @@ import (
 	"github.com/openshift/osde2e-common/internal/cmd"
 )
 
+const (
+	fedRampRolesCount    = 4 // HCP roles are disabled for FedRamp
+	commercialRolesCount = 7
+)
+
 // accountRoles represents all roles for a given prefix/version
 type accountRoles struct {
 	controlPlaneRoleARN string
-	installerRoleARN    string
+	InstallerRoleARN    string
```

Comment:
Can you remind me why we need to export this specific one only? I don't mind, just want to understand the reasoning